ui/src/utils/global-dialog.js

```diff
@@ -109,6 +109,8 @@ export default function (DefaultComponent) {
       }
     }
 
+    Vue.observable(props); 
```

Comment:
https://vuejs.org/v2/api/#Vue-observable

> In Vue 2.x, `Vue.observable` directly mutates the object passed to it, so that it is equivalent to the object returned, as [demonstrated here](https://vuejs.org/v2/guide/instance.html#Data-and-Methods). In Vue 3.x, a reactive proxy will be returned instead, leaving the original object non-reactive if mutated directly. Therefore, for future compatibility, we recommend always working with the object returned by `Vue.observable`, rather than the object originally passed to it.